web3swift/src/Account/EthereumAccount.swift

```diff
@@ -23,7 +23,7 @@ protocol EthereumAccountProtocol {
     func sign(hex: String) throws -> Data
     func sign(message: Data) throws -> Data
     func sign(message: String) throws -> Data
-    func sign(_ transaction: EthereumTransaction) throws -> SignedTransaction
+    func sign(transaction: EthereumTransaction) throws -> SignedTransaction
```

Comment:
Need to update the unit tests as currently failing with this API change

Reply:
Fixed in d172f935a67fd39c3359adc40f6bb037d05383a5
(not sure if there are tests outside of EthereumAccount+SignTransactionTests.swift)